qfieldsync/core/layer.py

```diff
@@ -74,6 +78,17 @@ def __init__(self, layer):
         self._is_geometry_locked = None
         self.read_layer()
 
+        self.storedInlocalizedDataPath = False
+        if self.layer.dataProvider() is not None:
+            pathResolver = QgsProject.instance().pathResolver()
+            md = QgsProviderRegistry.instance().providerMetadata(self.layer.dataProvider().name())
```

Comment:
not a strong opinion here, but I would personally call all these variables `meta` or `metadata`, instead of `md`.

Reply:
Couldn't we call it "markdown" ? :stuck_out_tongue_winking_eye: